clicommand/agent_start.go

```diff
@@ -954,16 +953,21 @@ var AgentStartCommand = cli.Command{
 			}
 
 			// Create an agent worker to run the agent
-			workers = append(workers,
-				agent.NewAgentWorker(
-					l.WithFields(logger.StringField("agent", ag.Name)), ag, mc, client, agent.AgentWorkerConfig{
-						AgentConfiguration: agentConf,
-						CancelSignal:       cancelSig,
-						Debug:              cfg.Debug,
-						DebugHTTP:          cfg.DebugHTTP,
-						SpawnIndex:         i,
-						AgentStdout:        os.Stdout,
-					}))
+			workers = append(workers, agent.NewAgentWorker(l.WithFields(
+				logger.StringField("agent", ag.Name)),
```

Comment:
The parentheses seem wrongly-aligned here. It looks a lot like most of this is contained within `l.WithFields(...)` but actually it's only `logger.StringField(...)` and the rest are args to `agent.NewAgentWorker`.

```suggestion
			workers = append(workers, agent.NewAgentWorker(
				l.WithFields(logger.StringField("agent", ag.Name)),
```